pkgs/applications/radio/openwebrx/default.nix

```
@@ -0,0 +1,89 @@
+{ stdenv, lib, buildPythonPackage, buildPythonApplication, fetchFromGitHub
+, pkg-config, cmake, setuptools
+, rtl-sdr, soapysdr-with-plugins, csdr, direwolf
+}:
+
+let
+
+  js8py = buildPythonPackage rec {
+    pname = "js8py";
+    version = "0.1.1";
+
+    src = fetchFromGitHub {
+      owner = "jketterl";
+      repo = pname;
+      rev = version;
+      sha256 = "1j80zclg1cl5clqd00qqa16prz7cyc32bvxqz2mh540cirygq24w";
+    };
+
```

Please add a pythonImportsCheck.

---

It already picks up the contained tests.

---

We changed over to always adding pythonImportsCheck  because it is easier for reviewers and it doesn't hurt to test that even if running tests. This gives us higher confidence when updating python package despite it being python packages.

```suggestion

    pythonImportsCheck = [ "js8py" ];

```